src/frontends/tensorflow/src/input_model.hpp

```diff
@@ -45,6 +48,7 @@ class InputModel : public ov::frontend::InputModel {
     void set_element_type(const ov::frontend::Place::Ptr& place, const ov::element::Type&) override;
     ov::element::Type get_element_type(const ov::frontend::Place::Ptr& place) const override;
     void set_tensor_value(const ov::frontend::Place::Ptr& place, const void* value) override;
+    std::shared_ptr<SMVariablesIndex> get_variables_index(void);
```

Comment:
please do not use `void` to be aligned with other code

Reply:
done